AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.java

```diff
@@ -2570,6 +2570,14 @@ public void showContextMenuDeckOptions() {
         }
     }
 
+    //Callback to open card browser for currently selected deck
+    public void openCardBrowser() { openCardBrowser(mContextMenuDid);}
```

Comment:
https://github.com/ankidroid/Anki-Android/wiki/Code-style#two-blank-lines-should-be-used-between-methods

```suggestion
    public void openCardBrowser() { 
        openCardBrowser(mContextMenuDid);
    }
    
    
```